scripts/generate_docs.sh

```diff
@@ -72,6 +72,8 @@ done
 
 for module in "${modules[@]}"; do
   args=("${jazzy_args[@]}"  --output "$root_path/docs/$version/$module" --module "$module")
+  args+=(--module-version $version
+         --root-url "https://apple.github.io/swift-nio/docs/$version/$module")
```

Comment:
overall looks good, i think this can be merged with the line above?

Reply:
done!